../gnome-maps-46.11/lib/maps-shield.c:1134:40: note: use function 'fabs' instead
        double angle = atan2 (x4 - x2, abs (y5 - radius - y1));

Index: lib/maps-shield.c
--- lib/maps-shield.c.orig
+++ lib/maps-shield.c
@@ -1176,7 +1176,7 @@ draw_shield (RenderCtx *ctx, double width, double heig
         double x6 = x8 - radius;
         double y1 = y0 + angle_sign * radius;
 
-        double angle = atan2 (x4 - x2, abs (y5 - radius - y1));
+        double angle = atan2 (x4 - x2, fabs (y5 - radius - y1));
         double sine = sin (angle);
         double cosine = cos (angle);
         double half_tangent = tan (angle / 2);
